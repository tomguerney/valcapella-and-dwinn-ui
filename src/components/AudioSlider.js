import React from 'react'
import Slider from '@material-ui/core/Slider';
import { withStyles } from '@material-ui/core/styles';

const AudioSlider = withStyles({
  root: {
    color: '#C8DEF4',
    height: 8,
    scrollBehavior: 'smooth',
    opacity: 1
  },
  thumb: {
    height: 24,
    width: 24,
    backgroundColor: '#fff',
    border: '2px solid currentColor',
    marginTop: -8,
    marginLeft: -12,
    '&:focus, &:hover, &$active': {
      boxShadow: 'inherit',
    },
  },
  active: {},
  track: {
    height: 8,
    borderRadius: 4,
  },
  rail: {
    opacity: 1,
    backgroundColor: '#9AA9CC',
    height: 8,
    borderRadius: 4,
  },
})(Slider);

export default AudioSlider
