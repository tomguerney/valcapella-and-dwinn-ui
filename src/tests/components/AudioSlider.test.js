import React from 'react'
import { shallow } from 'enzyme'
import AudioSlider from '../../components/AudioSlider'

test('should render AudioSlider correctly', () => {
  const wrapper = shallow(<AudioSlider />)
  expect(wrapper).toMatchSnapshot()
})