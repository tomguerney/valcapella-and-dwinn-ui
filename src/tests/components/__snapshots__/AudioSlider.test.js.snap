// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AudioSlider correctly 1`] = `
<WithStyles(ForwardRef(Slider))
  classes={
    Object {
      "active": "WithStyles(ForwardRef(Slider))-active-3",
      "rail": "WithStyles(ForwardRef(Slider))-rail-5",
      "root": "WithStyles(ForwardRef(Slider))-root-1",
      "thumb": "WithStyles(ForwardRef(Slider))-thumb-2",
      "track": "WithStyles(ForwardRef(Slider))-track-4",
    }
  }
/>
`;
